### DISCUSSION TITLE
Handling db migrations

### DISCUSSION THREAD

**Discussion by jerometremblay** [palkan/logidze#234] [Q&A] [answered]
Is there a recommended way to handle db migrations?

For example, I added a new field named "description". On a row, I set the value of this field to "initial description". Then I change the value again to "modified description".

When directly load the latest version of the object, i get "modified description" as expected.
When I load archived versions that were created while manipulating .description using .at(time), i get the correct versioned values.

However, when I load archived versions that were created before the migration, i get the current ("modified description") for all of them.

I managed to fix that (I think) by manually adding "description": null (the default column value) into the initial snapshot inside log_data.

Is that the recommended way to go? That sounds dangerous.

**Comment by palkan** [comment, marked as answer]
Please, take a look at this issue: https://github.com/palkan/logidze/issues/232

We have a workaround idea (see the comments), which we plan to implement.
